app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -60,6 +60,15 @@ type Mutation {
     addProductsToWishlist(wishlistId: ID!, wishlistItems: [WishlistItemInput!]!): AddProductsToWishlistOutput @doc(description: "Adds one or more products to the specified wish list. This mutation supports all product types") @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\AddProductsToWishlist")
     removeProductsFromWishlist(wishlistId: ID!, wishlistItemsIds: [ID!]!): RemoveProductsFromWishlistOutput @doc(description: "Removes one or more products from the specified wish list") @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\RemoveProductsFromWishlist")
     updateProductsInWishlist(wishlistId: ID!, wishlistItems: [WishlistItemUpdateInput!]!): UpdateProductsInWishlistOutput  @doc(description: "Updates one or more products in the specified wish list") @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\UpdateProductsInWishlist")
+    addWishlistItemsToCart(
+        wishlistId: ID!, @doc(description: "The unique ID of the requisition list")
```

Comment:
This should be "The unique ID of the wish list"

Reply:
hi @dthampy thanks for review
I have updated `wishlistUid` and `wishlistItemUids`  to  `wishlistId` and `wishlistItemIds` as per change request by @prabhuram93 
https://github.com/magento/magento2/pull/31584#discussion_r561961283